addons/binding/org.openhab.binding.sleepiq/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
Yes, we usually commit it if it exists - but normally, it isn't required if there are no project specific settings.